packages/Main/src/Source/VpcSource.js

```diff
@@ -0,0 +1,97 @@
+import LASParser from 'Parser/LASParser';
+import Fetcher from 'Provider/Fetcher';
+import Source from 'Source/Source';
+import { CopcSource, EntwinePointTileSource } from 'Main';
+
+/**
+ * An object defining the source of Entwine Point Tile data. It fetches and
+ * parses the main configuration file of Entwine Point Tile format,
+ * [`ept.json`](https://entwine.io/entwine-point-tile.html#ept-json).
+ *
+ * @extends Source
+ *
+ * @property {boolean} isEntwinePointTileSource - Used to checkout whether this
+ * source is a EntwinePointTileSource. Default is true. You should not change
+ * this, as it is used internally for optimisation.
+ * @property {string} url - The URL of the directory containing the whole
+ * Entwine Point Tile structure.
+ */
+class VpcSource extends Source {
+    /**
+     * @param {Object} config - The configuration, see {@link Source} for
+     * available values.
+     * @param {number|string} [config.colorDepth='auto'] - Does the color
+     * encoding is known ? Is it `8` or `16` bits ? By default it is to
+     * `'auto'`, but it will be more performant if a specific value is set.
```

Comment:
If I'm not wrong it is not set to `auto` by default but to `16` instead. 
Also, why would it be more performant ? Do you mean it might not work if not provided instead ?